Assets/MixedRealityToolkit/Interfaces/SceneSystem/IMixedRealitySceneSystem.cs

```diff
@@ -0,0 +1,245 @@
+    ///
+    /// The Manager scene is loaded first and remains loaded for the duration of the app.
+    /// Only one Manager scene is ever loaded, and no scene operation will ever unload it.
+    ///
+    /// The Lighting scene is a largely empty scene which controls lighting settings.
+    /// Ambient lighting, skybox, sun direction, etc. A default lighting scene is loaded on initialization.
+    /// After that the active lighting scene may be changed at any time via SetLightingScene.
+    /// Only one lighting scene can ever be loaded at a time.
+    ///
+    /// Content scenes are everything else. These can be loaded and unloaded at will in any combination.
+    ///
+    /// The scene actions provided improve on unity's SceneManagement events by ensuring that scenes
+    /// are considered valid before the action is invoked.
+    /// </summary>
+    public interface IMixedRealitySceneSystem : IMixedRealityEventSystem, IMixedRealityEventSource, IMixedRealityDataProvider
+    {
+        #region Actions
+
+        /// <summary>
+        /// Called just before a set of content scenes is loaded.
+        /// Includes names of all scenes about to be loaded.
+        /// </summary>
+        Action<IEnumerable<string>> OnWillLoadContent { get; set; }
+
+        /// <summary>
+        /// Called when a set of content scenes have been loaded, activated and are valid.
+        /// Includes names of all scenes loaded.
+        /// </summary>
+        Action<IEnumerable<string>> OnContentLoaded { get; set; }
+
+
+        /// <summary>
+        /// Called just before a set of content scenes will be unloaded.
+        /// Includes names of all scenes about to be unloaded.
+        /// </summary>
+        Action<IEnumerable<string>> OnWillUnloadContent { get; set; }
+
+        /// <summary>
+        /// Called after a set of content scenes have been completely unloaded.
+        /// Includes names of all scenes about to be unloaded.
+        /// </summary>
+        Action<IEnumerable<string>> OnContentUnloaded { get; set; }
+
+        /// <summary>
+        /// Called just before lighting scene is loaded.
+        /// Includes name of scene.
+        /// </summary>
+        Action<string> OnWillLoadLighting { get; set; }
+
+        /// <summary>
+        /// Called when a lighting scene has been loaded, activated and is valid.
+        /// Includes scene name.
+        /// </summary>
+        Action<string> OnLightingLoaded { get; set; }
+
+        /// <summary>
+        /// Called just before a lighting scene unload operation begins.
+        /// Includes scene name.
+        /// </summary>
+        Action<string> OnWillUnloadLighting { get; set; }
+
+        /// <summary>
+        /// Called after a lighting scene has been completely unloaded.
+        /// Includes scene name.
+        /// </summary>
+        Action<string> OnLightingUnloaded { get; set; }
+
+        /// <summary>
+        /// Called just prior to a scene being loaded.
```

Comment:
```suggestion
        /// Called just before to a scene being loaded.
```
This is a nit, but adds consistency in the docs :)